nixos/modules/services/web-apps/glitchtip.nix

```
@@ -154,37 +178,37 @@ in
   };
 
   config = lib.mkIf cfg.enable {
-    services.glitchtip.settings = {
-      DEBUG = lib.mkDefault 0;
-      DEBUG_TOOLBAR = lib.mkDefault 0;
-      PYTHONPATH = "${python.pkgs.makePythonPath pkg.propagatedBuildInputs}:${pkg}/lib/glitchtip";
-      DATABASE_URL = lib.mkIf cfg.database.createLocally "postgresql://@/glitchtip";
-      REDIS_URL = lib.mkIf cfg.redis.createLocally "unix://${config.services.redis.servers.glitchtip.unixSocket}";
-      CELERY_BROKER_URL = lib.mkIf cfg.redis.createLocally "redis+socket://${config.services.redis.servers.glitchtip.unixSocket}";
-      GLITCHTIP_VERSION = pkg.version;
+    warnings =
+      lib.optional (cfg.nginx.createLocally -> (cfg.listenAddress == "127.0.0.1" || cfg.port == "8000"))
```

```suggestion
      lib.optional (cfg.nginx.enable && (cfg.listenAddress != "127.0.0.1" || cfg.port != "8000"))
```

iiuc the warning should appear if `nginx.enable` is `true` and `listenAddress` or `port` deviate from their defaults (i.e. they are set explicitly)